### PR TITLE
Ruby on Rails capstone project - Budget app

### PR DESCRIPTION
**Pull request by hamid-murambiwa**
In this project, I was required to build a mobile web application where users can manage their budgets.
I made the following changes:
- Set up the repository and tools
- Create the models according to the ERD diagram
- Validations and tests
- Implement the authentication and authorization.
- Create the page to add a category.
- Create the home page.
- Create the page to add a transaction.
- Create the transactions page.
- Deploy the project, and test for final details.
- Record a video for the project.
- Create a README.md file

### CODE REVIEW

**Comment by omar-labana** [comment]
@hamid-murambiwa man, what is that VSCode theme :"D

**Comment by hamid-murambiwa** [comment]
> @hamid-murambiwa man, what is that VSCode theme :"D

Kimbie Dark, I think

File: README.md [hunk 1]
```diff
@@ -1 +1,98 @@
...
+>Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+>Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- This project built in Microverse's Fourth Module's Capstone.
+
```
Comment (omar-labana) [inline README.md:95]: - [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/)[](https://github.com/hamid-murambiwa/Budget-App/pull/5/files#live-version).

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @hamid-murambiwa, 

Excellent work so far! 👌
You have implemented almost all the required features correctly and without any bugs or errors. ✔️ 
Your repo is quite professional and the README is neatly formatted.
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
- [ ] Kindly, try to match the font found in the design guidelines(just a free similar one if it's paid) as instructed in the project guidelines.
- [ ] _Please, check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
## Hi @hamid-murambiwa,

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations🎉


### Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**